app/code/Magento/Braintree/view/frontend/web/js/view/payment/3d-secure.js

```diff
@@ -157,9 +157,9 @@ define([
                             context.paymentPayload.nonce = payload.nonce;
                             self.state.resolve();
                         } else {
-                            fullScreenLoader.stopLoader();
                             self.state.reject($t('Please try again with another form of payment.'));
                         }
+                        fullScreenLoader.stopLoader();
                     });
                 })
                 .fail(function () {
```

Comment:
What are your thoughts on adding an `.always(function () {  fullScreenLoader.stopLoader(); });` after this `.fail` to replace the `fullScreenLoader.stopLoader()` in every conditional path?

Reply:
Yes, I think that's a good way to refactor this part of code.
Thank you for the suggestion. I will update my PR now.